runtime/bootstrap.php

```diff
@@ -40,3 +41,10 @@ function juliantojd(int $month, int $day, int $year): int
         return Julian::juliantojd($month, $day, $year);
     }
 }
+
+if (!function_exists('cal_to_jd')) {
+    function cal_to_jd($year = null)
```

Comment:
```suggestion
    function cal_to_jd(int $calendar, int $month, int $day, int $year): int
```

Reply:
Good catch, I should try to add tests for this kind of things.